### DISCUSSION TITLE
Chrome 124

### DISCUSSION THREAD

**Discussion by krausest** [krausest/js-framework-benchmark#1670] [Announcements]
I'm really happy to publish [results](https://krausest.github.io/js-framework-benchmark/2024/table_chrome_124.0.6367.91.html) for chrome 124. Actually I didn't believe in getting #1655 fixed, but thanks to @robbiespeed we found the cause.
If you want some details: Seems like window.gc() doesn't clean up all the dom nodes in every case for chrome 124 and somehow those frameworks gained some speed from that (ivi was one of those).
We're using Chrome for Testing from this release on such that we don't have the performance impact of security fixes.

Furthermore we optimized the vanillajs implementations and @syduki added a new vanillajs implementation.

Attached you find a binary build of all frameworks such that you can run the benchmark without installing and building all frameworks (see README for how to use the build.zip)

<hr /><em>This discussion was created from the release <a href='https://github.com/krausest/js-framework-benchmark/releases/tag/'>Chrome 124</a>.</em>

**Comment by antonmak1** [comment]
The situation was, of course, difficult with version 124. I hope there will be fewer such "jokes" in the next versions of Google's Chrome. Although, who knows :-)
